website/docs/r/msk_cluster.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout: "aws"
+page_title: "AWS: aws_msk_cluster"
+sidebar_current: "docs-aws-resource-msk-cluster"
+description: |-
+  Terraform resource for managing an AWS Managed Streaming for Kafka cluster
+---
+
+# aws_msk_cluster
+
+Manages AWS Managed Streaming for Kafka cluster
+
```

Comment:
_Maintainer Note:_ Adding the preview callout for this resource documentation

~> **NOTE:** This AWS service is in Preview and may change before General Availability release. Backwards compatibility is not guaranteed between Terraform AWS Provider releases.